BranchMetrics/cpp-branch-deep-linking-attribution
# SDK-447 -- Full Windows Sample App

## Reference
SDK-447 -- Full Windows Sample App

## Description
Having "Hello World" will get us only so far.  We want to have a full-blown Windows Sample that shows all of the functionality of the C++ SDK

- Include Deep Linking
- Include Attribution

This is the "Pick A Color" application, as it will then be compatible with Android's HelloBranch.

## Testing Instructions
* Sample app should compile and run
* Using the links in the README, the colors should change as appropriate.

I tested both the Win32 and Win64 versions.

## Risk Assessment [`LOW`]
There is no change to the Core SDK, this is just a sample application.


## Reviewer Checklist (To be checked off by the reviewer only)

- Correctness & Style
